static/app/utils/profiling/colors/utils.tsx

```diff
@@ -114,11 +111,11 @@ export function toRGBAString(r: number, g: number, b: number, alpha: number): st
   )}, ${alpha})`;
 }
 
-export function defaultFrameSortKey(frame: Frame): string {
-  return (frame.file || '') + frame.name;
+export function defaultFrameSortKey(frame: FlamegraphFrame): string {
+  return frame.frame.name + (frame.frame.file || '');
```

Comment:
Why did name move before file?

Reply:
We always have a name, but we dont always have the file/binary, so I wanted to have that out first